portfolio_optimizer/anaconda-project.yml

```
@@ -8,21 +8,19 @@ labels:
 - hvplot
 - holoviews
 
-user_fields: [labels, skip, maintainers]
+user_fields: [labels, skip, maintainers, user_fields, user_fields, user_fields, user_fields]
```

This is from an anaconda-project bug, which I thought had been fixed, but maybe you have an older version? If it's the current anaconda-project version please file a bug report.

```suggestion
user_fields: [labels, skip, maintainers]
```